nautobot_netbox_importer/tests/test_import.py

```diff
@@ -50,6 +52,10 @@ def setUp(self):
         """Set up test environment."""
         super().setUp()
 
+        # pylint: disable=invalid-name
+        Status = apps.get_model("extras", "Status")
+        Status.objects.all().delete()
```

Comment:
Is this to avoid conflict with the default Status objects? If so, note that you may want to do the same thing for `Role` objects as well.

Reply:
Correct, so far there are no issues with `Role`, would do that once necessary.